containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -178,6 +179,46 @@ public void addServletHandlers(String serverHostName) {
             rewriteHandler.addRule(disableKeepAliveRule);
         }
         
+        // Security: Respond the "Expect-CT: max-age=31536000, report-uri=..." header
+
+        HeaderPatternRule expectCtRule = new HeaderPatternRule();
+        expectCtRule.setPattern("/*");
+        expectCtRule.setName("Expect-CT");
+        expectCtRule.setValue("max-age=31536000, report-uri=\"http://csp.yahoo.com/beacon/csp?src=yahoocom-expect-ct-report-only\"");
+        rewriteHandler.addRule(expectCtRule);
+
+        // Security: Respond the "Strict-Transport-Security: max-age=31536000" header
+
+        HeaderPatternRule hstsRule = new HeaderPatternRule();
+        hstsRule.setPattern("/*");
+        hstsRule.setName(HttpHeader.STRICT_TRANSPORT_SECURITY.asString());
+        hstsRule.setValue("max-age=31536000");
+        rewriteHandler.addRule(hstsRule);
+
+        // Security: Respond the "X-Content-Type-Options: nosniff" header (see https://docs.google.com/document/d/1ovG_qSqpbS1q2OpEX1Jo08w8dVdi4un1JT7CLz5unX4/edit)
+
+        HeaderPatternRule noSnifRule = new HeaderPatternRule();
+        noSnifRule.setPattern("/*");
+        noSnifRule.setName("X-Content-Type-Options");
+        noSnifRule.setValue("nosniff");
+        rewriteHandler.addRule(noSnifRule);
+
+        // Security: Respond the "Referrer-Policy: strict-origin-when-cross-origin" header (see https://docs.google.com/document/d/1ovG_qSqpbS1q2OpEX1Jo08w8dVdi4un1JT7CLz5unX4/edit)
+
+        HeaderPatternRule referrerPolicyRule = new HeaderPatternRule();
+        referrerPolicyRule.setPattern("/*");
+        referrerPolicyRule.setName("Referrer-Policy");
+        referrerPolicyRule.setValue("strict-origin-when-cross-origin");
+        rewriteHandler.addRule(referrerPolicyRule);
+
+        // Security: Respond the "Cache-Control: must-revalidate,no-cache,no-store" header (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control)
+
+        HeaderPatternRule cacheControlRule = new HeaderPatternRule();
+        cacheControlRule.setPattern("/*");
+        cacheControlRule.setName(HttpHeader.CACHE_CONTROL.asString());
+        cacheControlRule.setValue("must-revalidate,no-cache,no-store");
```

Comment:
it might make sense to make this value configurable. in case someone is running some other proxy service in front of zms and wants to cache results. we can use the value specified above as default.